src/NepaliDate.ts

```diff
@@ -642,6 +643,26 @@ class NepaliDate {
         )
         return NepaliDate.fromEnglishDate(year, month0, day, hour, minute, second, ms)
     }
+
+    /**
+     * Returns the number of days in a specific month of a given year.
+     *
+     * @param year - The year to fetch the month from.
+     * @param month0 - The month to get the number of days for.
+     * @returns The number of days in the specified month.
+     * @throws {Error} If the year or month is out of range.
+     */
+    static getDaysOfMonth(year: number, month0: number): number {
```

Comment:
```suggestion
    static getDaysOfMonth(year: number, month: number): number {
```
the variable can be named month

Reply:
Hmm. We have been using `month0` on the public-facing functions so they won't get confused using 0 as the first month. 
But I think it's better to use `month` everywhere. I will create an issue for this refactoring.
Thanks for your suggestion.

Reply:
Your suggestion is incomplete. Will have to create a new commit.

Reply:
Updated